app/dashboard/helpers.py

```diff
@@ -465,6 +475,17 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
             logging.error(f'{e} encountered during new bounty creation for: {url}')
             new_bounty = None
 
+        # notify a user that a bounty has been reserved for them once the bounty has been created
+        if reserved_for_user and new_bounty is not None:
+            from_email = _('founders@gitcoin.co')
+            email_subject = _('Reserved Issue')
```

Comment:
This whole block should be moved to the `emails.py` file in it's own email method.